src/black/__init__.py

```diff
@@ -181,6 +186,15 @@ def read_pyproject_toml(
     return value
 
 
+def spellcheck_pyproject_toml_keys(
+    ctx: click.Context, config_keys: List[str], config_file_path: str
+) -> None:
+    available_config_options = {param.name for param in ctx.command.params}
+    for key in config_keys:
+        if key not in available_config_options:
+            raise InvalidConfigKey(f"Invalid key {key} in {config_file_path}")
```

Comment:
If there are multiple invalid keys, could you make the error message list them all at once? That makes for a better debugging experience, since otherwise if you have N invalid keys you have to run Black N times to see them all.

Reply:
Agreed, just updated PR.